abcFinance/abcfinanceagent.py

```diff
@@ -0,0 +1,19 @@
+import abcEconomics
+
+class AbcFinanceAgent(abcEconomics.Agent):
+    def __init__(self, *param, **kwparam):
+        super().__init__(*param, **kwparam)
+
+        self.accounts = Ledger(
+            residual_account_name=kwparam.get('residual_account_name', 'equity'))
+
+    def __getattr__(self, attribute):
+        if attribute in ['book_end_of_period', 'book', 'make_stock_accounts', 'make_flow_accounts',
```

Comment:
What is the reasoning in limiting the attributes? I think it is common for any agent that inherits from an `AbcFinanceAgent` to have additional attributes.

Reply:
hmm, no idea, this was old code... I'm happy to change the Agent (Accountant?) class